includes/crons-rate-limits.mdx

```diff
@@ -0,0 +1,5 @@
+## Rate Limits
+
+Crons imposes a rate limit on check-ins to prevent abuse and resource overuse. Specifically, you can only send a maximum of **6 check-ins per minute per existing monitor environment**. This limit is enforced on a per-project basis, meaning that the rate limit applies collectively to all monitor environments within a given project. You can check if any of your check-ins are being dropped in the [Usage Stats](/product/stats/#usage-stats) page.
```

Comment:
Is it anywhere specific on the usage stats page? Can we add a small section in that doc to explain where to find cron check ins?